UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/NMDataService.cs

```diff
@@ -382,27 +382,27 @@ public async Task<byte[]> DownloadFssFileAsync(string batchId, string fileName,
 
         }
 
-        public async Task<byte[]> DownloadFSSZipFileAsync(string batchId, string fileName, string correlationId)
+        public async Task<byte[]> DownloadFSSZipFileAsync(string batchId, string fileName, bool isDistributorUser, string correlationId)
```

Comment:
No need to pass this isDistributorUser parameter from controller as distributor token can be used for downloading both partner & public zip files. Please revert this change.
```suggestion
        public async Task<byte[]> DownloadFSSZipFileAsync(string batchId, string fileName, string correlationId)
```

Reply:
Done